jinja2/qfdmo/_addresses_partials/filters/carte_apropos.html

```diff
@@ -0,0 +1,68 @@
+<!-- Panel «Légende» -->
+<div class='fr-container qfdmo-px-0 qfdmo-absolute qfdmo-inset-0 qfdmo-font-black qfdmo-z-[10001] qfdmo-flex qfdmo-flex-col qfdmo-hidden'
```

Comment:
Je suis toujours un peu vigilant sur les incrémentations de un en un sur les z-index.
Si on rajoute un layer, on peut se retrouver bloqué. 

J'aime bien incrémenter de 10 en 10, ça évite de tout reprendre le moment venu. 

```suggestion
<div class='fr-container qfdmo-px-0 qfdmo-absolute qfdmo-inset-0 qfdmo-font-black qfdmo-z-[10010] qfdmo-flex qfdmo-flex-col qfdmo-hidden'
```

Reply:
Fait pour tous les z-index > 10000